junit-jupiter-engine/src/test/java/org/junit/jupiter/api/extension/ExtensionComposabilityTests.java

```diff
@@ -53,18 +52,20 @@ void ensureJupiterExtensionApisAreComposable() {
 
 		List<String> expectedMethodSignatures = expectedMethods.stream()
 				.map(this::methodSignature)
+				.filter(name -> !name.contains("jacoco"))
```

Comment:
Are those jacoco methods by any chance "synthetic"?

If so, please simply filter out synthetic methods instead of checking if the method name contains "jacoco".

In any case, please go ahead and push this to `master` since it is not related to this PR and should be adopted ASAP anyway.